### ISSUE TITLE
[no-duplicate-attributes] - Allow the combination of the "map binding" and "static value" use of the `style` and `class` attributes

### ISSUE THREAD

**Issue by mbriand-lucca** [angular-eslint/angular-eslint#1406]
**Description and reproduction of the issue**

The rule `no-duplicate-attributes` should allow the combination of the "map binding" and "static value" use of the `style` and `class` attributes. According to the Angular documentation (https://angular.io/guide/style-precedence#style-precedence), Angular merges both attributes which means their combined use is valid.

```JSON
{
  "rules": {
    "@angular-eslint/template/no-duplicate-attributes":  "error"
  }
}
```

```HTML
// Valid :
<div class="bar" [class]="dynamic"></div>
<div style="color: blue" [style]="styleExpression"></div>

// Invalid :
<div class="bar" class="dynamic"></div>
<div style="color: blue" style="background-color: red"></div>
```

**Suggested Change**

Allow the combination of the "map binding" and "static value" use of the `style` and `class` attributes.

**Versions**

| package                                  | version   |
| ---------------------------------------- | --------- |
| `@angular-eslint/eslint-plugin-template` | `15.2.1`  |
| `@angular-eslint/template-parser`        | `15.2.1`  |
| `@typescript-eslint/parser`              | `5.57.1`  |
| `ESLint`                                 | `8.37.0`  |
| `node`                                   | `18.12.1` |

![image](https://github.com/angular-eslint/angular-eslint/assets/111852435/bf2639ea-1152-43b1-85b2-85751dadeaf8)
